sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/reactive/AbstractReactiveDatabaseIT.java

```diff
@@ -0,0 +1,243 @@
+        Assertions.assertEquals(HttpStatus.SC_CREATED, creation.statusCode());
+        String result = given()
+                .when().get("library/books/author/Vern")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .extract().body().asString();
+        Assertions.assertEquals("[Around_the_World_in_Eighty_Days]", result);
+    }
+
+    @Test
+    public void deleteAuthor() {
+        given().delete("library/author/1")
+                .then()
+                .statusCode(HttpStatus.SC_NO_CONTENT);
+        Response response = given()
+                .when().get("/library/author/1");
+        Assertions.assertEquals(HttpStatus.SC_NOT_FOUND, response.statusCode());
+    }
+
+    @Test
+    public void useDTO() {
+        Response response = given().when().get("library/dto/4");
+        Assertions.assertEquals(HttpStatus.SC_OK, response.statusCode());
+        JsonPath jsonPath = response.getBody().jsonPath();
+        Assertions.assertEquals("Thinking fast and slow", jsonPath.getString("[0].title"));
+        Assertions.assertEquals("Attention and Effort", jsonPath.getString("[1].title"));
+    }
+
+    @Test
+    public void useSession() {
+        String title = given()
+                .when().get("/hibernate/books/2")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .extract().body().asString();
+        Assertions.assertEquals("Thinking fast and slow", title);
+    }
+
+    @Test
+    public void useTransaction() {
+        Response creation = given()
+                .contentType(ContentType.JSON)
+                .post("hibernate/books/Dick/Ubik");
+        Assertions.assertEquals(HttpStatus.SC_CREATED, creation.statusCode());
+        Response response = given()
+                .when().get("/library/books/author/Dick");
+        Assertions.assertEquals(HttpStatus.SC_OK, response.statusCode());
+        Assertions.assertEquals("[Ubik]", response.getBody().asString());
+    }
+
+    @Test
+    public void convertValue() {
+        Response response = getApp().given().get("/library/isbn/2");
+        Assertions.assertEquals(HttpStatus.SC_OK, response.statusCode());
+        Assertions.assertEquals("9780374275631", response.body().asString());
+    }
+
+    @Test
+    public void convertZeroValue() {
+        Response response = getApp().given().get("/library/isbn/3");
+        Assertions.assertEquals(HttpStatus.SC_OK, response.statusCode());
+        Assertions.assertEquals("0", response.body().asString());
+    }
+
+    @Test
+    public void setConvertedZeroValue() {
+        Response change = getApp().given().put("/library/isbn/1/5170261586");
+        Assertions.assertEquals(HttpStatus.SC_OK, change.statusCode());
+        Response lookup = getApp().given().get("/library/isbn/1");
+        Assertions.assertEquals(HttpStatus.SC_OK, lookup.statusCode());
+        Assertions.assertEquals("5170261586", lookup.body().asString());
+    }
```

Comment:
Add test to check the raw value in the database (basically after conversion). ++ Drop ConverterTest and reflect it's logic in these IT tests

Reply:
Implemented